src/balena-init.sql

```diff
@@ -18,6 +18,8 @@ ALTER COLUMN "api heartbeat state" SET DEFAULT 'unknown';
 CREATE INDEX IF NOT EXISTS "api_key_actor_idx"
 ON "api key" ("is of-actor");
 
+CREATE INDEX IF NOT EXISTS "application_organization_idx"
+ON "application" ("organization");
```

Comment:
NIT: Could you put this below the actor index please, it feels weird not to have actor be first in the list due to its specialness